addons/csw/functions/fnc_deployWeapon.sqf

```diff
@@ -0,0 +1,63 @@
+/*
+ * Author: TCVM
+ * Deploys the current CSW
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call ace_csw_fnc_deployWeapon
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params["_tripod", "_player"];
+
+[{
+    params["_tripod", "_player"];
+    private _carryWeaponClassname = secondaryWeapon _player;
+    _player removeWeaponGlobal _carryWeaponClassname;
```

Comment:
Same as above, secondary weapon magazines may need to be handled in this file as well if `removeWeaponGlobal` removes those.

Reply:
Again, because of how the CSW's are implemented ammo in the secondaryWeapon should not be an issue. I clarified how I implemented them in the edited PR fourth paragraph.